presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -122,6 +125,12 @@ public void setUp()
         finalizerService.start();
     }
 
+    @BeforeMethod
+    public void init()
+    {
+        nodeTaskMap = new NodeTaskMap(finalizerService);
```

Comment:
you might be able to extract more variables into `@BeforeMethod`
this seems to be working locally for me. 
```
    @BeforeMethod
    public void init()
    {
        plan = createPlan();
        nodeTaskMap = new NodeTaskMap(finalizerService);
        stage = createSqlStageExecution(plan, nodeTaskMap);
    }
```

Reply:
@ajaygeorge I tried looking into that and found that the `plan` and subsequently the `stage` variables are used differently depending on the test. For example, `testBalancedSplitAssignment()` has two created plans therefore leaving the `plan` and 
`stage` variables to be discarded, having initialized them fruitlessly.